### PR TITLE
Fix Colorblindness

### PR DESCRIPTION
**Pull request by Arokha**
Does what it says!

### CODE REVIEW

**Comment by SilverTalismen** [comment]
Yaay!

**Comment by SpadesNeil** [comment]
-attempts to review code-
https://www.youtube.com/watch?v=KufiqCh3Qd4
-defers to Leshana-

File: code/modules/mob/living/living.dm [hunk 1]
```diff
@@ -1124,6 +1124,9 @@ default behaviour is:
 	var/list/colors_to_blend = list()
 	for(var/datum/modifier/M in modifiers)
 		if(!isnull(M.client_color))
+			if(islist(M.client_color)) //It's a color matrix! Forget it. Just use that one.
```
Comment (Leshana) [inline code/modules/mob/living/living.dm]: ok!

File: code/modules/resleeving/infocore_records.dm [hunk 2]
```diff
@@ -76,6 +76,7 @@
 	var/body_oocnotes
 	var/list/limb_data = list(BP_HEAD, BP_L_HAND, BP_R_HAND, BP_L_ARM, BP_R_ARM, BP_L_FOOT, BP_R_FOOT, BP_L_LEG, BP_R_LEG, BP_GROIN, BP_TORSO)
 	var/list/organ_data = list(O_HEART, O_EYES, O_LUNGS, O_BRAIN)
+	var/list/genetic_modifiers = list()
```
Comment (Leshana) [inline code/modules/resleeving/infocore_records.dm]: oh!

**Comment by Anewbe** [comment]
Did this ever get sent our way?

**Pull request merged**